examples/nlp/bidirectional_lstm_imdb.py

```diff
@@ -41,12 +41,13 @@
 )
 print(len(x_train), "Training sequences")
 print(len(x_val), "Validation sequences")
+# pad train and validation data to consider only 200 words in each movie review
```

Comment:
```
# Use pad_sequence to standardize sequence length:
# this will truncate sequences longer than 200 words and zero-pad sequences shorter than 200 words.
```